SU2_CFD/src/output/filewriter/CSurfaceFVMDataSorter.cpp

```diff
@@ -1119,7 +1133,31 @@ void CSurfaceFVMDataSorter::SortConnectivity(CConfig *config, CGeometry *geometr
 
 }
 
-void CSurfaceFVMDataSorter::SortSurfaceConnectivity(CConfig *config, CGeometry *geometry, unsigned short Elem_Type) {
+void CSurfaceFVMDataSorter::SortConnectivity(CConfig *config, CGeometry *geometry, vector<string> markerList) {
```

Comment:
Another vector that should be passed by const& here and in SortSurfaceConnectivity.